### DISCUSSION TITLE
Create mod now allowed to place items inside players claim.

### DISCUSSION THREAD

**Discussion by Eriador26** [GriefPrevention/GriefPrevention#1380] [Q&A]
Hello Hello,

So We are running a Mohist server which allows plugins AND mods. This way we can keep the regular plugin files running but also add some modded in the mix.

We figured out that the Create Mod that we have got an item that is placing items for a player. For example the Deplyer of the createmod places sapplings. Though it seems that the mod is not allowed to place items inside the claim. Outside claims it works fine.

Any way how I can adress this issue and have machines of set players work? or does this whole Plugin + Mod become to difficult for the plugin itself?

Lemme know.

**Comment by Jikoo** [comment]
Create likely uses a fake player to place objects. You'd need to whitelist that fake player.

**Reply by Eriador26** [reply]
Do you have a way to add this to griefprevention or how to create this whitelist?
Cause so far All i find is people telling eachother to whitelist set fake player but no one is explaining HOW.

**Reply by Jikoo** [reply]
Unfortunately core GP verifies players' existence before adding them to the trust list. Either the server implementation would need to translate fake players to offline players for the purpose of Bukkit plugin compatibility or an addon could bypass GP's checks and directly add the UUID to the trust list without verification.

**Comment by RoboMWM** [comment]
Duplicate of https://github.com/TechFortress/GriefPrevention/discussions/1305
